math_.py

```diff
@@ -13,3 +13,14 @@ def substract(a: int, b: int) -> int:
 def list_substract(a: List, b: List) -> List:
     assert len(a)==len(b), "len(a) != len(b)"
     return [i-j for i, j in zip(a,b)]
+
+def isnum(a: Any):
+    if isinstance(a, int):
+        return True
+    elif isinstance(a, str):
+        try:
+            _ = int(a)
+            return True
+        except Exception:
```

Comment:
There is no newline at the end of the file. Adding a newline at the end of the file can improve readability and prevent potential issues with some text editors.